tutoring.md

```diff
@@ -0,0 +1,57 @@
+
+The current schedule for Open Tutoring is:
+
+| Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday |
+|:------:|:-------:|:---------:|:--------:|:------:|:--------:|:------:|
+|        | 5-7 PM  |           | 5-7 PM   |        | 11 - 2PM |        |
+
+(Last updated 2/23/2016 - Check your class Google calendar for temporary schedule changes.)
+
+### Ad-hoc Tutoring
+Ad-hoc Tutoring is an ideal option when you feel that more dedicated 1-on-1 instruction is necessary, or you're unable to attend the Open Tutoring hours.
+
+Ad-hoc Tutoring consists of one-time, individual tutoring sessions arranged through our mailing list (the `ada-tutors` Google group). The mailing list has many volunteers from the tech industry in Seattle, including a number of Adies from previous cohorts.
+
+When you post a message to the mailing list please include the following information to help our tutors figure out if they would be a good fit for working with you:
+- Where you are at in the curriculum, including what project you're working on if your tutoring request relates to a specific project
+- What particular technologies you're working with (Ruby, Rails, JavaScript, Backbone, etc.)
+- What times would work best for you
+- Something about yourself and/or your preferred learning and communication styles
+
+Once you've posted a message to the mailing list an Ada staff member will follow up with you within a few days to make sure that you were able to find assistance or help to make other arrangements.
+
+### Scheduled Tutoring
+Scheduled Tutoring is generally reserved for students who have need of recurrent, dedicated instruction outside of class such as when significant instruction material has been missed due to absences.
+
+Because our resources for this service are more limited than the other services, Scheduled Tutoring must be arranged with the approval of Ada staff. If you feel that the above situation applies to you, please reach out to your instructor to discuss the possibility of setting up Scheduled Tutoring.
+
+Scheduled Tutoring involves agreed-upon Areas of Focus and associated goals which will be evaluated at the completion of the tutoring schedule to determine if it was successful or if additional tutoring may be needed. Tutoring schedules are generally for 2-3 weeks with 1 or 2 sessions per week, as determined by Ada staff.
+
+## FAQ
+### Which tutoring service should I use?
+Because of the differing availability of tutoring services we strongly advise that all students interested in tutoring first attend Open Tutoring or, if that is not possible, request dedicated Ad-hoc Tutoring through the mailing list.
+
+If both of those options are not available or are unsuccessful we suggest that you speak with one of your instructors and a more individual tutoring plan can be devised.
```

Comment:
Maybe something about what they should do to prepare for a scheduled tutoring or open tutoring session, make a list of problems/questions etc.